### PR TITLE
Diagnostic bundle to include containerizer debug endpoint tracking da…

### PR DESCRIPTION
**Pull request by abudnik**
…ta for stuck tasks.

## High-level description

Added agent's `/containerizer/debug` endpoint to the diag bundle.

## Corresponding DC/OS tickets (required)

  - [DCOS-56373](https://jira.mesosphere.com/browse/DCOS-56373) Diagnostic bundle to include containerizer debug endpoint tracking data for stuck tasks.

## Related tickets (optional)

  - [DCOS-55383](https://jira.mesosphere.com/browse/DCOS-55383) Containerizer debug endpoint (epic).

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [x] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

File: packages/dcos-integration-test/extra/test_dcos_diagnostics.py [hunk 1]
```diff
@@ -478,6 +478,7 @@ def _download_bundle_from_master(dcos_api_session, master_index, bundle):
 
     expected_agent_common_files = [
         '5051-containers.json',
+        '5051-containerizer_debug.json'
```
Comment (janisz) [inline packages/dcos-integration-test/extra/test_dcos_diagnostics.py]:
```suggestion
        '5051-containerizer_debug.json',
```

**Comment by abudnik** [comment]
@mesosphere-mergebot label Ready For Review

File: CHANGES.md [hunk 2]
```diff
@@ -60,6 +60,8 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 
 * Add dcos-net overlay metrics. (DCOS_OSS-5324)
 
+* Diagnostic bundle to include containerizer debug endpoint tracking data for stuck tasks. [DCOS-55383]
```
Comment (qianzhangxa) [inline CHANGES.md]: s/[DCOS-55383]/(DCOS-55383)/

File: CHANGES.md [hunk 3]
```diff
@@ -60,6 +60,8 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 
 * Add dcos-net overlay metrics. (DCOS_OSS-5324)
 
+* Diagnostic bundle to include containerizer debug endpoint tracking data for stuck tasks. (DCOS-55383)
```
Comment (jgehrcke) [inline CHANGES.md]: Mhm I'd love for this sentence to be clearer. Not blocking the PR though.

**Comment by janisz** [comment]
@mesosphere-mergebot sync

**Pull request merged**